packages/grafana-runtime/src/services/SidecarContext_EXPERIMENTAL.ts

```diff
@@ -21,17 +21,20 @@ export function useSidecar_EXPERIMENTAL() {
     throw new Error('No SidecarContext found');
   }
 
-  const activePluginId = useObservable(service.activePluginIdObservable, service.activePluginId);
   const initialContext = useObservable(service.initialContextObservable, service.initialContext);
+  const activePluginId = useObservable(service.activePluginIdObservable, service.activePluginId);
+  const locationService = service.getLocationService();
 
   return {
     activePluginId,
     initialContext,
+    locationService,
     // TODO: currently this allows anybody to open any app, in the future we should probably scope this to the
     //  current app but that means we will need to incorporate this better into the plugin platform APIs which
     //  we will do once the functionality is reasonably stable
-    openApp: (pluginId: string) => service.openApp(pluginId),
-    closeApp: (pluginId: string) => service.closeApp(pluginId),
+    openApp: (pluginId: string, context?: unknown) => service.openApp(pluginId, context),
```

Comment:
I think if you define the service methods as arrow functions, you can then simplify this code with `openApp: service.openApp` or does that not work?

Reply:
Ah yeah good point. I think I did some changes to args before but now this mapping isn't needed.

Reply:
Uh sorry I did the change and then forgot to push it upstream before merge 🤦  will update it in the next PR